app/client/packages/rts/src/ctl/verify-migration.mjs

```
@@ -0,0 +1,221 @@
+    
+    const pgValue = pgDoc[key.toLowerCase()]; // PostgreSQL fields are lowercase
+    
+    if (!isEquivalent(mongoValue, pgValue)) {
+      differences.push(`Field '${key}' mismatch - Mongo: ${mongoValue}, Postgres: ${pgValue}`);
+    }
+  }
+  
+  // Check for extra fields in PostgreSQL
+  for (const [key, pgValue] of Object.entries(pgDoc)) {
+    const mongoKey = key.toLowerCase();
+    if (!mongoDoc.hasOwnProperty(mongoKey) && pgValue !== null) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)